src/index.html

```diff
@@ -17,7 +17,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     
     <link rel="icon" href="/images/icon.png">
-    <link rel="apple-touch-icon" href="/images/icon.png">
+    <link rel="apple-touch-icon" href="/images/icon-bg.png">
```

Comment:
Let's check if this one is working, it's a png with white bg, I would like to test it.